.circleci/config.yml

```diff
@@ -36,6 +36,9 @@ jobs:
       - run:
           name: Wait for SQS and SNS to be ready
           command: curl -4 --connect-timeout 30 --retry-connrefused --retry 20 --retry-delay 5 http://localhost:4566
+      - run:
+          name: Wait for Postgres to be ready
+          command:  netstat -na | grep '5432 '
```

Comment:
Oh that's not going to work is it :( need to add some retries.  Although if it takes longer to start than localstack something's wrong.